src/js_native_api.h

```diff
@@ -446,7 +446,18 @@ NAPI_EXTERN napi_status napi_reject_deferred(napi_env env,
 NAPI_EXTERN napi_status napi_is_promise(napi_env env,
                                         napi_value value,
                                         bool* is_promise);
-
+#if NAPI_VERSION >= 8
+NAPI_EXTERN napi_status napi_promise_then(napi_env env,
```

Comment:
A new Node api funcitons need to be added as Experimental first and then promoted when we are comfortable with them being added to a Node-api version.

They would need to be guarded by #ifdef NAPI_EXPERIMENTAL

Reply:
ok，thank you。i will do it well.